topic-operator/pom.xml

```diff
@@ -117,11 +117,6 @@
             <artifactId>mockito-core</artifactId>
             <scope>test</scope>
         </dependency>
-        <dependency>
-            <groupId>org.mockito</groupId>
-            <artifactId>mockito-inline</artifactId>
-            <scope>test</scope>
-        </dependency>
```

Comment:
is this mockito removal related to the Java 21 update. Just curious.

Reply:
Yes. Java 21 needs a new version of Mockito ... and in a new version of Mockito, `mockito-inline` does not exist anymore (it does not seem to be missing in any tests, so either it was merged into other Mockito artifact or was unused anyway).